python/hail/dataset.py

```diff
@@ -5746,17 +5813,18 @@ def trio_matrix(self, pedigree, complete_trios=False):
         are the sample IDs of the trio probands. The column annotations and
         entries of the matrix are changed in the following ways:
 
-        The new column annotation schema is a ``Struct`` with three ``Struct`` children
-        (``proband``, ``father``, and ``mother``),
-        each with an ``id`` and ``annotations`` field. The schema of each ``annotations``
-        field is the column annotation schema of the input dataset.
+        The new column annotation schema is a ``Struct`` with five children
+        (structs ``proband``, ``father``, and ``mother``, boolean ``sex``, and string ``famID``).
```

Comment:
I think ``sex`` here is ``isFemale``, right?

Reply:
good catch.